src/generator/passes.jl

```diff
@@ -802,12 +802,12 @@ function (x::FunctionPrinter)(dag::ExprDAG, options::Dict)
     general_options = get(options, "general", Dict())
     log_options = get(general_options, "log", Dict())
     show_info = get(log_options, "FunctionPrinter_log", x.show_info)
-    blacklist = get(general_options, "printer_blacklist", [])
+    denylist = get(general_options, "printer_denylist", [])
```

Comment:
```suggestion
    denylist = get(general_options, "printer_denylist", get(general_options, "printer_blacklist", []))
```
This would be a simple, local strategy to continue supporting the old names while allowing projects to transition to the new names. A specific deprecation could be added in the future, but at least this avoids throwing the unwelcoming flag of forcing other projects to write new dependencies on this identifier.